packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -272,53 +272,59 @@ export default class MuiDetailGenerator implements DetailGenerator {
     index?: number
   ): ts.SourceFile {
 
-    // const inputElementCode = this.createInputElementFromTemplate(property);
-    // if(inputElementCode) {
-    //   const inputElementAst = createAst(inputElementCode);
-    //   if(inputElementAst) {
-    //     const inputElement = findByCondition<ts.JsxChild>(inputElementAst, (node: ts.Node) => {
-    //       return  ts.isJsxText(node) || ts.isJsxExpression(node) || ts.isJsxElement(node) || ts.isJsxSelfClosingElement(node) || ts.isJsxFragment(node);
-    //     });
+    const inputElementCode = this.createInputElementFromTemplate(property);
+
+    if(inputElementCode) {
+      const inputElementAst = createAst(inputElementCode);
+
+      if(inputElementAst) {
+        const inputElement = findByCondition<ts.JsxChild>(inputElementAst, (node: ts.Node) => {
+          return  ts.isJsxText(node) || ts.isJsxExpression(node) || ts.isJsxElement(node) || ts.isJsxSelfClosingElement(node) || ts.isJsxFragment(node);
+        });
         
-    //     if(inputElement) {
-    //       const gridItemElement = this.createGridItemElement([inputElement]);
-
-    //       let newElements: ts.JsxElement[];
-    //       if(index && index < gridElements.length + 1){
-    //         newElements = [...gridElements.slice(0, index - 1), gridItemElement, ...gridElements.slice(index - 1)]
-    //       }else{
-    //         newElements = [...gridElements, gridItemElement]
-    //       }
+        if(inputElement) {          
+          clearNodePosition(inputElement);
+          
+          const gridItemElement = this.createGridItemElement([inputElement]);
+
+          let newElements: ts.JsxElement[];
```

Comment:
please move 6 lines to a helper function for example `insertJsxElement(existingElements, newElement, index)`